.cirrus.yml

```diff
@@ -154,10 +158,6 @@ unit_task:
     only_if: &not_build_docs >-
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' &&
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:BUILD.*'
-    depends_on: &smoke_vendor_cross
-      - smoke
-      - vendor
-      - cross_build
```

Comment:
I wonder if we should at least keep the dependency for vendor & smoke test as those are fast enough.

In podman we have basically two stages build/validate -> actual tests:
https://raw.githubusercontent.com/wiki/containers/podman/cirrus-map.svg

So I think it makes sense for buildah to follow that, unless we really like to safe the the 7 mins form the smoke test?

Reply:
Having added depends_on bits back, it looks like we're getting just about everything else stalled until the cross-build check finishes.  Replacing some of the "only_if" directives with "skip" directives looks like it will avoid this weirdness.